### PR TITLE
Netlifycms example

### PR DESCRIPTION
**Pull request by tsgriff**
React-Static Basic with NetlifyCMS example

### CODE REVIEW

File: examples/netlifycms/src/containers/Post.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import Moment from 'react-moment'
+//
+
+export default withRouteData(({ post }) => (
+  <div className="blog-post">
+    <Link to="/blog/">{'<'} Back</Link>
+    <br />
+    <h3>{post.data.title}</h3>
+    <Moment format="MMMM Do, YYYY">{post.data.date}</Moment>
+    <img className="image" src={require(`../images/uploads/${post.data.thumbnail}`)} alt="" />
```
Comment (tannerlinsley) [inline examples/netlifycms/src/containers/Post.js]: This is likely going to need to change to reference the `public` directory. Like this:
```javascript
<img className="image" src={`/images/uploads/${post.data.thumbnail}`)} alt="" />
```

**Comment by tannerlinsley** [comment]
You rock!

**Comment by tannerlinsley** [comment]
I'll be releasing this with my changes by tonight.

**Comment by tsgriff** [comment]
Awesome! Thanks, Tanner. I'm honored to be a part of this project :)

**Pull request merged**